frontend/src/app/projects/create-project/create-project.component.html

```diff
@@ -53,16 +53,33 @@ <h3 class="flex justify-center sm:hidden">Create new project</h3>
                   formControlName="visibility"
                   class="flex flex-col"
                 >
-                  <mat-radio-button
-                    *ngFor="
-                      let visibility of projectService.getAvailableVisibilities()
-                    "
-                    [value]="visibility"
-                  >
-                    {{
-                      projectService.getProjectVisibilityDescription(visibility)
-                    }}
-                  </mat-radio-button>
+                  @for (
+                    visibility of projectService.getAvailableVisibilities();
+                    track visibility
+                  ) {
+                    <mat-radio-button [value]="visibility">
+                      {{
+                        projectService.getProjectVisibilityDescription(
+                          visibility
+                        )
+                      }}
+                    </mat-radio-button>
+                  }
+                </mat-radio-group>
+              </div>
+            </fieldset>
+            <fieldset>
+              <div class="flex flex-col">
+                <legend>Project type</legend>
```

Comment:
![image](https://github.com/user-attachments/assets/99b59586-5844-4b52-9cca-436cf793a023)

The gap between the title and the radios looks a bit compressed. This is due to the description of General in the brackets, which expands to two lines. 

I'd either reduce the length of the description, e. g., by removing the "multiple" word. Or alternatively add the description always in a new line with smaller text and dark grey color.